### PR TITLE
fix(ci): Fix cargo deny syntax

### PR DESCRIPTION
**Pull request by teor2345**
## Motivation

There's a copy-paste error in our cargo deny CI, but it still seems to work?

## Review

This is a low priority CI fix.

### Reviewer Checklist

  - [x] Will the PR name make sense to users?
    - [ ] Does it need extra CHANGELOG info? (new features, breaking changes, large changes)
  - [x] Are the PR labels correct?
  - [x] Does the code do what the ticket and PR says?
  - [ ] How do you know it works? Does it have tests?

### CODE REVIEW

File: .github/workflows/continous-integration-os.yml [hunk 1]
```diff
@@ -270,4 +270,4 @@ jobs:
         uses: EmbarkStudios/cargo-deny-action@v1
         with:
           command: check ${{ matrix.checks }}
-          args: --workspace with features ${{ matrix.features }}
+          args: --workspace ${{ matrix.features }}
```
Comment (arya2) [inline .github/workflows/continous-integration-os.yml]:
```suggestion
          arguments: --workspace ${{ matrix.features }}
```

**Review by arya2 — COMMENTED** [review]
Nice catch!

> it still seems to work?

It seems to ignore `args` altogether.

File: .github/workflows/continous-integration-os.yml (hunk 1 above)
Comment (teor2345) [reply .github/workflows/continous-integration-os.yml]: Yep, that's the correct syntax:
https://github.com/EmbarkStudios/cargo-deny-action#use-specific-rust-version

**Review by arya2 — APPROVED** [review]
🎉

**Pull request merged**